addons/web_editor/static/src/js/backend/fields.js

```diff
@@ -60,7 +60,8 @@ var FieldTextHtmlSimple = basic_fields.DebouncedField.extend(TranslatableFieldMi
      * @override
      */
     isSet: function () {
-        return this.value && this.value !== "<p><br/></p>" && this.value.match(/\S/);
+        // Remove inline style with regex because when if field has style-inline option then <p> element css is apply in element
+        return this.value && this.value.replace(/ style="([^"]*)"|&nbsp;|\s/g,'') !== "<p><br></p>" && this.value.match(/\S/);
```

Comment:
@tde-banana-odoo 

- Comment should be changed to proper english
- Regex can be simplified
- If regex removes whitespaces, matching against non-whitespaces afterwards is non-efficient/readable for nothing
- JS conventions

Something like this: 
```javascript
var value = this.value && this.value.replace(/ style="[^"]*"|&nbsp;|\s/g, '');
return value && value !== "" && value !== '<p><br/></p>';
```
?
Anyways, that should be @ged-odoo 's team that checks that kind of stuff now :wink: